enhancements/sig-policy/28-selective-policy-enforcment/README.md

```diff
@@ -0,0 +1,603 @@
+  kind: Policy
+  name: test-policy-1
+```
+
+### Action override in PlacementBinding
+The `remediationActionOverride.remediationAction` field allows users
+to override the remediationAction specified in the bound policy. Valid
+values for this field include unset (no override) and `enforce`.
+
+The behavior of this field is:
+
++ An `enforce` override sets the remediationAction of the bound Policy
+  to `enforce` for all clusters bound by this PlacementBinding.
++ The behavior of multiple PlacementBindings for the same policy and
+  cluster remains unchanged (they are additive).
+  + For reducing resources the implementation should de-duplicate any
+    child policies created
++ A policy bound to the same cluster under multiple PlacementBindings
+  will be set to `enforce` if _any_ of those bindings has
+  `remediationActionOverride.remediationAction` set to enforce.
++ A policy with `remediationAction` already set to `enforce` is
+  unaffected by a PlacementBinding with
+  `remediationActionOverride.remediationAction` set to enforce.
+
+### Sub filtering option in PlacementBinding
+The `remediationActionOverride.remediationAction` field allows users
+to control what clusters can be selected for a
+remediationActionOverride. This field is a boolean with valid values
+of `remediationActionOverride.subFilter: [false|true]`. To preserve
+the existing semantics of multiple placement bindings attached to a
+single policy this option defaults to `false`.
+
+The behavior of the `remediationActionOverride` when the `subFilter`
+is `true` is:
+
++ For the bound Policy, only clusters selected by another
+  PlacementBinding where `subFilter` is false will be considered for
+  `remediationAction` override.
+  + A cluster must be bound by both this `subFilter: true` binding
+    _and_ at least one `subFilter: false` PlacementBinding for the
+    override to be applied.
+
+The subFilter option is specified to allow a user to ensure that a
+single PlacementBinding can be used to define the set of potentially
+affected clusters. Without the subFilter option, it is possible to
+erroneously select additional clusters in the "override"
+PlacementBinding and accidentally enforce the policy to those
+additional clusters. The `subFilter` option, when set true, ensures
+that a second PlacementBinding used to enforce the policy to a smaller
+number of clusters will respect the intent of the initial binding.
+
+The behavior of the `remediationActionOverride` when the `subFilter`
+is `false` is:
+
++ For the bound Policy, all selected clusters will have the
+  `remediationAction` overridden.
+
+`true` this option restricts the set of clusters considered for a
+remediationActionOverride. When true, the set of clusters that are
+considered when evaluating the PlacementRule for the specific
+PlacementBinding is restricted to the set that are selected by all
+other PlacementBindings for the policy where the `subFilter` field is
+false (examples below).
+
```

Comment:
I think it should be mentioned that the enhancement will work as expected regardless of whether the PlacementBinding is attached to a Policy or to a PolicySet.

Reply:
I updated the proposal to call out that it applies to PolicySet as well. Thanks.